### DISCUSSION TITLE
Background image doesn't appear inside circular progress bar

### DISCUSSION THREAD

**Discussion by donacrisante** [neuefische/questions#732] [Web] [answered]
Hello everyone,
I have been trying since this morning to integrate an image as background in my circular progress bar, but without success. I have tried different ways. Either the picture doesn't appear, or the background is black, as it is now. Can someone maybe help me with this? The commented out lines are part of my attempts from this morning.

Many thanks in advance!

Here is my PR:
https://github.com/donacrisante/Capstone-project/pull/17

**Comment by donacrisante** [comment]
I have also tried the method with CircularProgressBarWithChildren:

[...]
import { CircularProgressbarWithChildren } from "react-circular-progressbar";

[...]

![Bildschirm­foto 2023-09-19 um 14 40 37](https://github.com/neuefische/questions/assets/128093782/ee8ee2b3-c46e-4bda-8831-ea5ef91bf895)

[...]

But the homepage displays this:

<img width="358" alt="Bildschirm­foto 2023-09-19 um 14 41 36" src="https://github.com/neuefische/questions/assets/128093782/7a0efcb4-b0de-4c88-8e72-46862608d03d">

The image is not found and I don't understand why.

**Comment by Kheirah** [comment, marked as answer]
Some things were right, some weren't. Let's start with the things that were right. The styled components for the background image can stay as it is:
```js
const StyledBackground = styled.div`
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background-image: url(/SustainableCity.jpg);
  background-size: cover;
  background-repeat: no-repeat;
  background-position: center;
  z-index: -1;
`; 
```
Actually that's about it. Now to the things that need adjusting. For implementing the ciruclar progress bar and the `StyledBackground` you write:
```js
<StyledCircularProgressbar
    value={co2Emission}
    maxValue={2000}
    text={`${co2Emission.toFixed(2)} kg`}
/>
<StyledBackground />
```
Additionally, you need to move the "SustainableCity.jpg" to `/public`. Delete the .js file that you previously created for the image. Delete the import of that .js file.
When you do all that you will get this:

<img width="206" alt="Screenshot 2023-09-20 102136" src="https://github.com/neuefische/questions/assets/22839911/4949c4d8-0f8e-46d2-bc62-08e4e86dd440">
